### PR TITLE
Add word movement granularities to android

### PR DESCRIPTION
**Pull request by jonahwilliams**
I image we can support word and line by looking for whitespace and newlines .... right?

### CODE REVIEW

**Comment by jonahwilliams** [comment]
Based on offline discussion, we will only include word for now since we don't have a way to support line/page/paragraph yet.

**Comment by goderbauer** [comment]
> I image we can support word and line by looking for whitespace and newlines .... right?

image

**Comment by goderbauer** [comment]
You'd have to use ICU data to correctly determine word/line/etc breaks: http://icu-project.org/apiref/icu4c/classicu_1_1BreakIterator.html

This is not fully exposed to dart land.

If you do, be aware of this bug: http://icu-project.org/apiref/icu4c/classicu_1_1BreakIterator.html

**Comment by jonahwilliams** [comment]
@goderbauer do you want to wait for that issue to be resolved before we add this to the framework?

File: shell/platform/android/io/flutter/view/AccessibilityBridge.java [hunk 1]
```diff
@@ -480,7 +490,17 @@ boolean performCursorMoveAction(
                     return true;
                 }
             }
-                // TODO(goderbauer): support other granularities.
+            case AccessibilityNodeInfo.MOVEMENT_GRANULARITY_WORD:
```
Comment (goderbauer) [inline shell/platform/android/io/flutter/view/AccessibilityBridge.java]: Do the case statements need to end with `break` to avoid fall-through?

**Comment by goderbauer** [comment]
No, I wouldn't wait.

**Pull request merged**